test/component2_suggestors/deprecated_lifecycle_suggestor_test.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:over_react_codemod/src/component2_suggestors'
+    '/deprecated_lifecycle_suggestor.dart';
+import 'package:test/test.dart';
+
+import '../util.dart';
+import '../../lib/src/component2_suggestors/component2_constants.dart';
```

Comment:
#nit there's a lint on this line saying to avoid relative imports for files in `/lib`
```suggestion
import 'package:over_react_codemod/src/component2_suggestors/component2_constants.dart';
```

Reply:
Good catch!